style.css

```diff
@@ -179,6 +177,154 @@ body {
   display: none;
 }
 
+.second {
+  background-color: #FAE8E4;
+  height: 900px;
+  width: 100%;
+}
+
+.second-container {
+  display: grid;
+  grid-template-columns: 730px 488px;
+  grid-template-rows: 593px; /*622*/
+  grid-column-gap: 42px;
+  padding-top: 172px;
+  padding-left: 106px;
+}
+
+.second-container__text-white {
+  width: 488px;
+  height: 556px;
+  background-color: #fff;
+  box-shadow: 8px 8px 25px rgba(122,122,122,0.5);
+}
+
+.second-container__text-white-container {
+  width: 378px;
+  height: 556px;
+  margin: 0 auto;
+}
+
+.second-container__text-title {
+  font-size: 40px;
+  line-height: 1;
+  margin: 0;
+  padding-top: 40px;
+}
+
+.second-container__bags {
+  padding-top: 60px;
+}
+
+.second-container__text-subtitle {
+  font-family: Montserrat;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 1.1;
+  letter-spacing: .07em;
+  opacity: .5;
+}
+
+.second-container__text-main {
+  font-family: Montserrat;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 1.4;
+  opacity: .5;
+  margin-top: 65px;
+}
+
+.second-container__text-color {
+  display: flex;
+  flex-direction: row;
+}
+
+.second-container__text-gray,
+.second-container__text-pink,
+.second-container__text-blue,
+.second-container__text-green {
+  width: 30.18px;
+  height: 30px;
+  border-radius: 50%;
+}
+
+.second-container__text-gray:hover,
+.second-container__text-pink:hover,
+.second-container__text-blue:hover,
+.second-container__text-green:hover {
+  transform: scale(1.2);
+  transition: 1s ease-out .2s;
+  cursor: pointer;
+}
+
+.second-container__text-gray {
+  background-color: #DEDEDE;
+
+}
+
+.second-container__text-pink {
```

Comment:
Codacy found an issue: [Expected selector ".second-container__text-pink" to come before selector ".second-container__text-pink:hover" (no-descending-specificity)](https://app.codacy.com/gh/WebHeroSchool/step3/pullRequest?prid=5898150)